src/containers/search-result-page/assets/js/script.js

```diff
@@ -42,23 +88,35 @@ function getMoviesFromAPI() {
         .then(function (data) {
 
             readMoviesArray(data.results);
+            console.log(queryURL)
```

Comment:
Is it necessary to leave `console.log`?

Reply:
absolutely not
all console.log should be deleted for final version.
Let me fix.

Reply:
I left console.log to check the URL.
Final version will be deleted.